blocks/textarea/textarea.css

```diff
@@ -7,12 +7,16 @@
   border-radius: 1px;
   outline: none;
   resize: none;
-  overflow: hidden;
-  font-family: "Open Sans", sans-serif; 
+  font-family: "OpenSans", arial, sans-serif; 
   color: rgb(40, 54, 69); 
   font-size: 18px;
-  font-weight: 400;
-  line-height: 30px;
+  line-height: 166.667%;
+}
+
+.form__textarea::placeholder {
```

Comment:
You have written styles for placeholder in `form__textarea`, but there is no textarea with this class name. Please, change it to
```
 .textarea::placeholder {
  Rest of your code
 }
```